index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Apple</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/016a3b4736.js"></script>
+</head>
+
+<body>
+  <header class="top-menu">
+    <nav class="nav-header">
+      <ul class="ul-nav-header">
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span></span>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <span>
+
+            </span>
+          </a>
+        </li>
+
+      </ul>
+      <div class="div-search">
+        <i class="fas fa-search"></i>
+        <input type="text" name="search" value="">
+      </div>
+    </nav>
+  </header>
+
+  <article class="">
+    <div class="main-article">
+      <header>
+        <div class="main-container">
+          <a href="#"><img id="main-img-1" src="./images/headertext.png" alt=""></a>
+          <a href="#"><img id="main-img-2" src="./images/ptext.png" alt=""></a>
+          <a href="#"><img id="main-img-3" src="./images/playtext.png" alt=""></a>
+        </div>
+      </header>
+    </div>
+  </article>
+  <aside class="promo">
+    <div class="promo-container">
+      <a href="#"> <img class="promo-image" src="./images/promo1.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo2.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo3.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo4.jpg" alt=""></a>
+    </div>
+  </aside>
```

Comment:
# Optional
You can wrap these sections inside a `main` tag, since they define the main content of the body.